content/swift/concepts/sets/terms/removeAll/removeAll.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: '.removeAll()' # 
+Description: 'Learn how to remove every item from a Set in Swift.' # 
+Subjects: # 
+  - 'iOS'
+  - 'Mobile Development'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'iOS'
+  - 'SwiftUI'
+  - 'Sets'
+  - 'Xcode'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **.removeAll()** method is used to remove every value from a [Set](https://www.codecademy.com/resources/docs/swift/sets) in Swift. 
+
+
+## Syntax
+
+```
+setName.removeAll()
+```  
+`.removeAll()` will remove all values from `setName`.
+
+## Example
+
+The following is an example of the .removeAll() method:
+
+<img width="706" alt="Screen Shot 2022-11-29 at 10 19 28 AM" src="https://user-images.githubusercontent.com/115716841/204600438-394b799b-a4de-4df7-b544-af19614cb9c6.png">
```

Comment:
Please use code blocks to illustrate examples, not images. the example code should be in a \`\`\`swift block and the output should be placed in a \`\`\`shell block.

Reply:
I recommend something like: 

## Example

The following is an example of the .removeAll() method:

```swift
var weather: Set = ["spring", "summer", "autumn", "fall", "winter"]

weather.removeAll()

print(weather)
```

This will outpit:

```shell
[]
```